index.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title></title>
+        <link rel="stylesheet" href="style.css">
+
+    </head>
+    <body>
+        <div id="page-wrapper">
```

Comment:
So please make sure that you use section tag instead of the div tag. Its because in HTML section is a semantic tag whereas div is not.